benchmarks/src/main/java/com/slack/kaldb/IndexingBenchmark.java

```diff
@@ -92,6 +100,64 @@ public void tearDown() throws IOException {
       walk.sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
     }
     registry.close();
+    if (reader != null) {
+      reader.close();
+    }
+  }
+
+  @Benchmark
+  public void measureAPILogIndexing() throws IOException {
```

Comment:
Can you please move this benchmark into it's own file and a slack internal package? That way, it will be much easier to remove this code, once we no longer need it?

Reply:
Isn't it the same as removing this method and some instance variables?

Reply:
True, but that is much harder than removing a class file.

Reply:
Also having it in the same file makes it easier for debug actions